### DISCUSSION TITLE
Welcome to addons Discussions!

### DISCUSSION THREAD

**Discussion by martinlevesque** [openode-io/addons#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rg03s** [comment]
Hello, one question, how can I add the mysql addon to my nodeJS app? I have created a folder called 'database' and inside I have created another called 'mysql' with the file config.json which is located in the addons repository but it doesn't seem to work. Thank you

**Reply by martinlevesque** [reply]
Hi!

Addons have been deprecated unfortunately, I will archive the repository. You can use an external service and use it for example via environment variables.
